README.md

```diff
@@ -162,6 +162,8 @@ models that are available out of the box.
 | claude                |     88.4 |       1.1 |
 | chatgpt               |     86.1 |       1.2 |
 | wizardlm-13b          |     75.3 |       1.5 |
+| airoboros-65b         |     73.9 |       1.5 |
+| airoboros-33b         |     73.3 |       1.6 |
```

Comment:
We're trying to keep the leaderboard readme small here, so only displaying results for the `minimal` model set. Could you please revert this change?